ili/utils/ndes_tf.py

```diff
@@ -30,8 +32,14 @@ def load_nde_pydelfi(
         model (str): model to use. 
             One of: mdn, maf.
         index (int, optional): index of the nde in the ensemble. Defaults to 0.
+        engine (str, optional): dummy argument to match sbi interface.
+            Must be set to 'NLE' or will be overwritten.
         **model_args: additional arguments to pass to the model.
     """
+    if 'NLE' not in engine:
+        logging.warning(f'Engine {engine} not supported in pydelfi backend. '
```

Comment:
As above, I think that this should raise an error rather than a warning, since it is safer to not guess what the user wants to do, but to just tell them that their request cannot be performed.

Reply:
Done